src/System.Security.Cryptography.X509Certificates/tests/CollectionExportTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public static class CollectionExportTests
+    {
+        private static X509Certificate2 CreateCert(string name)
+        {
+            var rsa = new RSACryptoServiceProvider(2048);
```

Comment:
`RSA.Create`.  Never `RSACryptoServiceProvider`.

Reply:
When the new test moves to the other file I'd just continue using the test certificates that the rest of the library does, though.

Reply:
> When the new test moves to the other file

I don't follow what you mean here?